log.js

```diff
@@ -0,0 +1,23 @@
+const winston = require('winston')
+
+const logger = winston.createLogger({
+    transports : [
+        new winston.transports.Console()
+    ]
+})    
+
+const output = (file) => {
+    const obj = {}
+    obj.log = (str) => {
```

Comment:
```
(...args) => logger.info(file, ...args)
```

Same with all the functions. and then you need `log.test.js`

Reply:
I jsut tested the first part. When I do `logger.info(file, args)`, it prints out `{"level": "info", "message": file}` and ignores `args`

> ```
> (...args) => logger.info(file, ...args)
> ```
> 
> Same with all the functions. and then you need `log.test.js`